app/presenters/appointment.rb

```diff
@@ -0,0 +1,5 @@
+class Appointment
+  def to_partial_path
+    '/confirmations/appointment'
```

Comment:
I think having this point at an empty template for now is fine, since we'll soon want to add content here.